### DISCUSSION TITLE
Redcap support in upcoming releases

### DISCUSSION THREAD

**Discussion by xfoukas** [srsran/srsRAN_Project#786] [General Help]
Hi,

Following up on the discussion in #148, has there been any change regarding Redcap? Are there any plans to add support in upcoming releases?

Thanks!

**Comment by pgawlowicz** [comment]
Unfortunately, there was no change in our current agenda. You can always check our Roadmap [here](https://docs.srsran.com/projects/project/en/latest/general/source/2_features_and_roadmap.html).
